bcda/web/api.go

```diff
@@ -33,37 +32,12 @@ import (
 
 var qc *que.Client
 
-/*
-  	swagger:route GET /api/v1/ExplanationOfBenefit/$export bulkData bulkEOBRequest
-
-	Start explanation of benefit data export
-
-	Initiates a job to collect explanation of benefit data from the Blue Button API for your ACO.
-
-	Produces:
-	- application/fhir+json
-
-	Security:
-		bearer_token:
-
-	Responses:
-		202: BulkRequestResponse
-		400: badRequestResponse
-		401: invalidCredentials
-		429: tooManyRequestsResponse
-		500: errorResponse
-*/
-
-func bulkEOBRequest(w http.ResponseWriter, r *http.Request) {
-	bulkRequest("ExplanationOfBenefit", w, r)
-}
-
 /*
 	swagger:route GET /api/v1/Patient/$export bulkData bulkPatientRequest
 
-	Start patient data export
+	Start patient, explanation of benefit, and coverage data export
```

Comment:
From Swagger, I am able to make a call to `/Patient`.  I think we should add the ability for the user to supply a `_type` parameter so that users can execute export jobs for individual resource types.  Does Swagger support that?